storehaus-hbase/src/main/scala/com/twitter/storehaus/asynchbase/AsyncHBaseStore.scala

```diff
@@ -20,7 +20,6 @@ import com.twitter.util.{Future, FuturePool}
 import java.util.concurrent.Executors
 import org.hbase.async.{DeleteRequest, PutRequest, GetRequest, HBaseClient}
 import com.twitter.bijection.{Injection, Codec, Conversion}
-import scala.Some
 import Conversion._
 import scala.collection.JavaConverters._
 
```

Comment:
actually, below I think we want to use `toFuture[T](d: Deferred[T]): Future[T]` and actually not use the `FuturePool` at all, I think. In fact, I think we could possibly remove the `futurePool` and the `threads` val.